culture/code-of-conduct.md

```diff
@@ -0,0 +1,124 @@
+
+Maliciousness towards other community members
+: deliberately attempting to make others feel bad, name-calling, singling out others for derision or exclusion. For example, telling someone they’re not a real programmer or that they don’t belong at 2i2c.
+
+Being especially unpleasant
+: for example, if we’ve received reports from multiple 2i2c users, team members, or collaborators of annoying, rude, or especially distracting behavior.
+
+## Scope
+
+2i2c community members are held to the standards outlined in this code of conduct when interacting in the 2i2c Slack or GitHub repositories, when interacting in-person at events where they could represent 2i2c (this is most professional events), in physical spaces with other 2i2c team members or collaborators, or in any {term}`ICSI` space.
+
+In addition, the 2i2c community and experience often extends outside those spaces—2i2c community members may go on walks together to get lunch, attend meetups or conferences as a group, communicate on social media, or interact with each other in other communities. Abusive or unwelcoming behavior between community members still has a profound impact on individuals and on the community when it happens beyond our walls. 2i2c Directors and the Steering Council will use our discretion when deciding whether to enforce this code of conduct and potentially remove someone from the 2i2c community after reports of such behavior happening outside of 2i2c, taking into account the impact on the individual community members involved as well as the impact on the community at large.
+
+The 2i2c Code of Conduct does not apply to users of any hubs that we manage on behalf of other organizations, though we encourage leaders in those communities to adopt a Code of Conduct for their hub infrastructure. The Code of Conduct **does** apply to any 2i2c Team Member that is acting in a community's hub.
+
+When in doubt, please {ref}`report <coc:reporting>` unacceptable behavior to us. If someone’s behavior outside of a 2i2c space makes you feel unsafe at 2i2c, that is absolutely relevant and actionable for us.
+
+
+## Enforcement
+
+We’ve categorized unacceptable behavior into abuse and unwelcoming behavior in the section above.
+
+If we witness or receive a report about abusive behavior, we will contact the perpetrator to have a conversation with them and verify what has transpired, and they will be removed from the 2i2c community. Their Slack account will be deactivated, and permissions will be removed from any 2i2c-related repositories. They will not be welcome in any physical or digital spaces covered by the 2i2c Code of Conduct.
```

Comment:
This makes it sound like removal of the perpetrator is the only possible outcome because it only discusses abusive behaviour and not unwelcoming behaviour as well. This is laid out nicely in the protocol doc, but if the CoC was all I read, it could easily be understood that the only possible outcome is removing someone (which might deter someone from submitting a CoC report on unwelcoming behaviour if they want to flag something, but not necessarily take drastic action). I think a short statement on unwelcoming behaviour here would solve that.